### DISCUSSION TITLE
Flex Vs PhysX Particle System

### DISCUSSION THREAD

**Discussion by rossetti73** [NVIDIA-Omniverse/PhysX#124] [Q&A]
Hi,
i thinking   to migrate my app from Flex to PhysX  Particle System . 

PhysX Particle System, which Advantage does it have compared to Flex?

Are all Flex features in PhysX Particle System? or same features less in  PhysX Particle System?

I see PhysX Particle System work only with CUDA but this for me is ok.

**Comment by sschirm** [comment]
Hi @rossetti73 , 
Roughly speaking, PhysX Particles have the advantage of interacting better with PhysX rigid bodies and other PhysX features.
One thing were PhysX Particles still lack is reporting for contacts and callbacks for forces, etc.

**Reply by rossetti73** [reply]
Hi, Thanks about answer .
I Start try to migrate part of my but find some difficult about parameter  in PhySx  have some correspondence with FlexParam ?
Exist some document to migrate from Flex to PhySx?
Until now result of clothing in PhySx seams Bad  sure  i wrong  parameter.

**Reply by sschirm** [reply]
Sorry we don't have a migration document. However you might want to try our next release because it includes some fixes of sever bugs in the cloth cooker.

**Reply by rossetti73** [reply]
> Sorry we don't have a migration document. However you might want to try our next release because it includes some fixes of sever bugs in the cloth cooker.

Sure i will try it .
When do you think  is available next release ?

**Reply by preist-nvidia** [reply]
The 5.2.0 release should be around May 24.

**Reply by rossetti73** [reply]
thanks
very good!!

**Reply by preist-nvidia** [reply]
@rossetti73 - FYI we are delaying the release a bit, my apologies. At the moment, we plan for end of June.

**Comment by rossetti73** [comment]
ohh bad news,  we hope for more improvement .......

Thank for update



e-mail: ***@***.***

P Please consider the environment before printing this e-mail


Il giorno ven 26 mag 2023 alle ore 09:04 Philipp Reist <
***@***.***> ha scritto:
